docs/src/main/asciidoc/_extending-wildfly/Example_subsystem.adoc

```diff
@@ -1415,7 +1369,7 @@ to install the deployers, which is done in the following method:
         //see SubDeploymentProcessor for explanation of the phases
         context.addStep(new AbstractDeploymentChainStep() {
             public void execute(DeploymentProcessorTarget processorTarget) {
-                processorTarget.addDeploymentProcessor(SubsystemDeploymentProcessor.PHASE, SubsystemDeploymentProcessor.priority, new SubsystemDeploymentProcessor());
+                processorTarget.addDeploymentProcessor(SubsystemExtension.SUBSYSTEM_NAME, SubsystemDeploymentProcessor.PHASE, SubsystemDeploymentProcessor.priority, new SubsystemDeploymentProcessor());
```

Comment:
Could you replace `SubsystemDeploymentProcessor.priority` with `SubsystemDeploymentProcessor.PRIORITY`?